src/3rdparty/walkontable/src/table/master.js

```diff
@@ -29,8 +29,11 @@ class MasterTable extends Table {
       const trimmingHeight = getStyle(trimmingElement, 'height', rootWindow);
       const trimmingOverflow = getStyle(trimmingElement, 'overflow', rootWindow);
       const holderStyle = this.holder.style;
-      const { scrollWidth, scrollHeight } = trimmingElement;
-      let { width, height } = trimmingElement.getBoundingClientRect();
+
+      const { scrollWidth, scrollHeight, offsetWidth, offsetHeight } = trimmingElement;
+      let width = offsetWidth;
+      let height = offsetHeight;
```

Comment:
I would change it to destructive assignment. For me it's more clear 🙂 But it's up to you.

```suggestion
let [width, height] = [offsetWidth, offsetHeight];
```

Reply:
I like it as is